dantenetto/spiderggj17
# added wave line renderer script and test scene for it

To test it:
 - open **WavesTestScene**
 - hit the "Fire1" button (just mouse click anywhere)
 - it should generate a wave for each click.

When positioning a wave, make sure to position its beginning outside the screen
- there are a bunch of configuration to play with, so don't be afraid to break it.

cleaning the project:
- if cleaning the project, remember to remove the script **TestWaveGenerator** in addition to removing the test scene.
